src/styles/main.scss

```diff
@@ -1 +1,72 @@
-// put code here
+body {
```

Comment:
Better to create separate `div` with class `.wrapper` or sth else because doesn't good set such classes to `body`